js/reload_controller.js

```diff
@@ -257,7 +325,6 @@ ReloadController.prototype.onInstall = function()
 ReloadController.prototype.reloadWindow = function(win, options = {})
 {
   chrome.tabs.getAllInWindow(win.id, (tabs) => {
-    const strategy = {}
```

Comment:
Please place that back in, the reason why it is needed is because it is used to carry on the reload strategies to the other tabs on right/left.